velocity-engine-core/src/test/java/org/apache/velocity/test/SecureIntrospectionTestCase.java

```diff
@@ -163,14 +166,35 @@ public void setProperty(String val)
     }
 
 
-	public Collection getCollection()
-	{
-		Collection c = new HashSet();
-		c.add("aaa");
-		c.add("bbb");
-		c.add("ccc");
-		return c;
-	}
+    public Collection getCollection()
+    {
+        Collection c = new HashSet();
+        c.add("aaa");
+        c.add("bbb");
+        c.add("ccc");
+        return c;
+    }
+
+    public ClassLoader getSampleClassLoader1()
+    {
+        return this.getClass().getClassLoader();
+    }
+
+    /**
+     * sample property which is a subclass of ClassLoader
+     * @return
+     */
+    public ClassLoader getSampleClassLoader2()
```

Comment:
are those two new methods being used somewhere/somehow ?

Reply:
Apparently not. I removed them.